shared/css/toc.css

```diff
@@ -0,0 +1,215 @@
+.standalone-room-wrapper .toc-wrapper {
```

Comment:
The file actually has it now.  I'm not sure when that got added, but we're all good here.